## T-test
### Tow-Sample
#### Paired
Each value of one group corresponds directly to a value in the other group; i.e. before and after value s after drug treatment for each individual patient.

Basically just perform a one-sample t-test

assume sample size is less than 30;
assume normal distribution??


#### Unpaired
The two populations are independent
H0: means of the two populations are equal

#### Unpaired Example

In [15]:
from statistics import *
import numpy as np
from scipy import stats

In [1]:
samples = [1,2,3,4,5]

In [4]:
mean(samples)

3

In [7]:
n1 = 12
n2 = 6

mean1 = 700
mean2 = 668

s1 = 21
s2 = 30

# test if mean1 is bigger than mean 2, alpha = 0.05

In [14]:
t = (mean1-mean2)/(np.sqrt(s1**2/n1+s2**2/n2))
t

2.3416375464467274

In [18]:
stats.t.ppf(0.975,16)

2.1199052992210112

In [20]:
stats.t.cdf(2.3,16)

0.98237724781747571

0.98 is bigger than 0.95, so reject

## Chi-Square Test

### For Goodness of Fit

What is a Chi-Square distribution?

Assumptions: Standard normally distributed variable (mean = 0, variance = 1)
    
When to use:

